input/Glfw/GlfwInputDevice.cpp

```diff
@@ -161,99 +177,83 @@ namespace NovelRT::Input::Glfw
         _availableKeys.emplace("Quote", NovelKey("Quote", GLFW_KEY_APOSTROPHE, GLFW_MOD_SHIFT));
         _availableKeys.emplace("Underscore", NovelKey("Underscore", GLFW_KEY_MINUS, GLFW_MOD_SHIFT));
         _availableKeys.emplace("Tilde", NovelKey("Tilde", GLFW_KEY_GRAVE_ACCENT, GLFW_MOD_SHIFT));
+#pragma endregion KeyMapping
 
         _isInitialised = true;
 
-        int width = 0;
-        int height = 0;
+        int32_t width = 0;
+        int32_t height = 0;
         glfwGetWindowSize(_window, &width, &height);
+        _windowDimensions = NovelRT::Maths::GeoVector2F(static_cast<float>(width), static_cast<float>(height));
+
         _logger.logInfo("GLFW input system initialised: window at {} x {}", width, height);
     }
 
     void GlfwInputDevice::Update(Timing::Timestamp /*delta*/)
     {
         double x = 0;
         double y = 0;
-        int width = 0;
-        int height = 0;
-        glfwGetCursorPos(_window, &x, &y);
+        int32_t width = 0;
+        int32_t height = 0;
         glfwGetWindowSize(_window, &width, &height);
-        _mousePos.x = static_cast<float>(x - (width / 2));
-        _mousePos.y = static_cast<float>(-y + (height / 2));
-
-        _previousStates = _mappedActions;
+        glfwGetCursorPos(_window, &x, &y);
+        _windowDimensions.x = static_cast<float>(width);
+        _windowDimensions.y =static_cast<float>(height);
+        //_mousePos = DetermineMouseScreenPosition(NovelRT::Maths::GeoVector2F(x, y));
```

Comment:
Dead code, please delete

Reply:
Should be removed with next commit pushed